src/react-components/src/components/02-molecules/Helpers/HelperErrorText.tsx

```diff
@@ -0,0 +1,31 @@
+/* MT-51 Helper and Error text*/
+import * as React from "react";
+import bem from "../../../utils/bem";
+
+export interface HelperErrorTextProps {
+  id: string;
+  isError: boolean;
+  modifiers?: string[];
+  blockName?: string;
+  baseClass?: string;
+  attributes?: {};
+}
+
+export default function HelperErrorText(props: React.PropsWithChildren<HelperErrorTextProps>) {
+  const { id, isError, attributes } = props;
+  let blockName = props.blockName ? props.blockName : "input-description";
+  let baseClass = props.baseClass ? props.baseClass : "form-item";
```

Comment:
These should be default props. You can assign them after the function declaration:

```
HelperErrorText.defaultProps = {
  blockName: "input-description",
  baseClass: "form-item",
};
```
and here destructure them: `const { id, isError, attributes, blockName, baseClass } = props;`.

Reply:
the difference between this and 
`  const { id, blockName = "input-description", baseClass = "form-item", isError, ...rest } = props;` is purely stylistic, right?